scripts/windows-setup.ps1

```diff
@@ -48,15 +48,14 @@ if (!$SkipTools) {
 
         Invoke-WebRequest -UseBasicParsing -Uri 'https://get.scoop.sh' `
             | Invoke-Expression
-        scoop bucket add versions
-        scoop install git go nvm python27
+        scoop install git go nvm python unzip
+        # Install and use latest node 16* version
+        nvm install 16
+        nvm list | Select-String '[16\.[0-9.]+]' | Select-Object -First 1 | ForEach-Object { $_.Matches.Value }
```

Comment:
Did you mean to run `nvm use` on the version number?

Reply:
oops, that was a mistake. I added `nvm use` to the command.